nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/dynamodb/DeleteDynamoDB.java

```diff
@@ -75,7 +75,9 @@
             Arrays.asList(TABLE, HASH_KEY_NAME, RANGE_KEY_NAME, HASH_KEY_VALUE, RANGE_KEY_VALUE,
                 HASH_KEY_VALUE_TYPE, RANGE_KEY_VALUE_TYPE, BATCH_SIZE, REGION, ACCESS_KEY, SECRET_KEY,
                 CREDENTIALS_FILE, AWS_CREDENTIALS_PROVIDER_SERVICE, TIMEOUT, SSL_CONTEXT_SERVICE,
-                PROXY_CONFIGURATION_SERVICE, PROXY_HOST, PROXY_HOST_PORT, PROXY_USERNAME, PROXY_PASSWORD));
+                PROXY_CONFIGURATION_SERVICE, PROXY_HOST, PROXY_HOST_PORT, PROXY_USERNAME, PROXY_PASSWORD,
```

Comment:
Since we need to change all concrete subclasses of the `AbstractAWSProcessor`, can't we gather the property descriptors defined in it at one place and use that instead of updating this list for each one of them.

I see that the `...DynamoDB` processors don't have the `ENDPOINT_OVERRIDE` included.
Is that intentional or a nice example of why a common gathering logic would be useful?

Reply:
That kind of refactor would probably be beyond the scope for this PR, maybe you can enter another jira for that work?

Reply:
Imo smaller refactors (like this) is best done while working on a change.
In this case particularly so, as the current change and the refactor is almost the same in scope and complexity.

Reply:
ENDPOINT_OVERRIDE was missing before. fixing it is out of the scope of this PR